cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -791,6 +837,38 @@ func (p *BicepProvider) purgeKeyVaults(
 	return nil
 }
 
+func (p *BicepProvider) purgeCognitiveAccounts(
+	ctx context.Context,
+	resourceGroup string,
+	cognitiveAccounts []armcognitiveservices.AccountsClientGetResponse,
+	options DestroyOptions,
+	skip bool,
+) error {
+	for _, cogAccount := range cognitiveAccounts {
+		accountName := cogAccount.Name
+		if accountName == nil {
+			return fmt.Errorf("Cognitive account without a name")
+		}
+		accountId := cogAccount.ID
+		if accountId == nil {
+			return fmt.Errorf("Cognitive account without an id")
+		}
+		location := cogAccount.Location
+		if location == nil {
+			return fmt.Errorf("Cognitive account without a location")
+		}
+
+		err := p.runPurgeAsStep(ctx, "Cognitive Account", *accountName, func() error {
```

Comment:
I see other resources are non-capitalized. Should we be consitent in capitalization here?